### PR TITLE
tpm2_nvread: Adding a warning to the user about max read size

### PR DESCRIPTION
**Pull request by idesai**
tpm2_nvread: Adding a warning to the user about max read size

### CODE REVIEW

File: tools/tpm2_nvread.c [hunk 1]
```diff
@@ -165,6 +165,11 @@ static bool init(int argc, char *argv[], tpm_nvread_ctx *ctx) {
                         optarg);
                 return false;
             }
+            if (ctx->size_to_read > 1024) {
```
Comment (williamcroberts) [inline tools/tpm2_nvread.c]: Couldn't we just keep reading in 1024 blocks for them?

Comment (williamcroberts) [inline tools/tpm2_nvread.c]: That seems to be what nv_write is doing...

**Comment by williamcroberts** [comment]
Better commit message, and a bug ticket on next would be nice.

File: tools/tpm2_nvread.c (hunk 1 above)
Comment (idesai) [inline tools/tpm2_nvread.c]: @williamcroberts correct, just need to test the nv index size prior with nv read public to avoid nv wear out.

**Comment by williamcroberts** [comment]
This will be on #363

**Pull request closed without merging**